### ISSUE TITLE
gradient backward question

### ISSUE THREAD

**Issue by yxmt-oss** [NVIDIA/MinkowskiEngine#578]
We want to utilize the gradient information of the coordinates in the SparseTensor type to generate adversarial samples. However, after training the model and performing backpropagation using the loss function, the gradient of the coordinates in the SparseTensor is None, while the gradient of the features is not None. How should we address this issue?
